### DISCUSSION TITLE
Wiki Update - Beginners Guide?

### DISCUSSION THREAD

**Discussion by ThePixyDon** [adultmm/AdultMediaManager#38] [Q&A]
Even though I've used Ember before, I'm finding using this module frustrating at the moment, I can't seem to get an automatic scrape to work.

I there a way to scrape for "Video" using specific Scrapers, rather than a "Movie" search fail then having to manually Switch to Video, choose Scraper then hit Search and then pick the movie?

Also is there a way to have the "Site" be used as the "Studio" tag?

An initial setup guide would be welcomed I believe.

**Comment by Sixpax** [comment]
If you are going to be doing a number of "video" rather than "movie" files, you could change it under settings > Movies > click on the AdultScraper, change the Video/movie to always search video first.

As to the site and studio, you can also look under "video settings" tab under AdultScraper and try ticking the "Add site name as studio".
